plotting/plot_window/base_qt_plot_window.py

```diff
@@ -45,10 +45,16 @@ def _start_draw_zoom_rect(self,mouse_event):
 
 
     def _redraw_zoom_rect(self,mouse_event):
+        cache = [self._new_axes_limits[0][:],self._new_axes_limits[1][:]]
         self._new_axes_limits[0][1] = mouse_event.xdata
         self._new_axes_limits[1][1] = mouse_event.ydata
         rect = zip(*self._new_axes_limits)
-        rect = self.axes.transData.transform(rect)
+        try:
```

Comment:
@EltayebAhmed I don't think that is acceptable unless there's absolutely on way around it.